source/learners/SFD_teams.rst

```diff
@@ -252,8 +252,7 @@ Although you can view discussions in any team, you must belong to a team to
 add new posts and responses to the team's discussion.
 
 Team discussions work in the same way as your course discussions. For
-information about course discussions, see :ref:`Anatomy of Open edX Course
-Discussions`.
+information about course discussions, see :ref:`Anatomy of Discussions`.
```

Comment:
This reference target (`Anatomy of Discussions`) isn't defined anywhere, so it's throwing an error.